src/dsp/loader.ts

```diff
@@ -18,7 +18,7 @@ export class AxHFDataLoader {
     options
   }: Readonly<{
     dataset: string;
-    split: 'train' | 'validation';
+    split: string;
```

Comment:
I noticed some traning sets split by a different value. Happy to revert this change if we want the split only to be 'train' | 'validation', but not sure the benefit we get here.